lib/sentry/logger_backend.ex

```diff
@@ -128,27 +97,54 @@ defmodule Sentry.LoggerBackend do
     :ok
   end
 
-  defp init(config, %__MODULE__{} = state) do
-    level = Keyword.get(config, :level, state.level)
+  defp log(level, msg, meta, state) do
+    sentry = meta[:sentry] || get_sentry_from_callers(meta[:callers] || [])
 
-    include_logger_metadata =
-      Keyword.get(config, :include_logger_metadata, state.include_logger_metadata)
+    opts =
+      [
+        event_source: :logger,
+        extra: %{logger_metadata: logger_metadata(meta, state), logger_level: level},
+        result: :none
+      ] ++ Map.to_list(sentry)
+
+    case meta[:crash_reason] do
+      {exception, stacktrace} when is_list(stacktrace) ->
+        Sentry.capture_exception(exception, [stacktrace: stacktrace] ++ opts)
+
+      reason when is_atom(reason) and not is_nil(reason) ->
+        Sentry.capture_exception(reason, opts)
```

Comment:
Do you know when this would ever be the case?

Reply:
yeah, I meant to put some details in the PR when I made these changes, but forgot.

A few of the cases don't match `%{__exception__: true}`. Specifically:

https://github.com/getsentry/sentry-elixir/pull/403/files#diff-c895426ce24017d1803054d94e4d71c3L41 crash_reason is:
`crash_reason: {{:bad_return_value, "I am throwing"}, []}`

https://github.com/getsentry/sentry-elixir/pull/403/files#diff-c895426ce24017d1803054d94e4d71c3L69 is:
`crash_reason: {:bad_exit, []}`

https://github.com/getsentry/sentry-elixir/pull/403/files#diff-c895426ce24017d1803054d94e4d71c3L155:
`crash_reason: {{:timeout,
    {GenServer, :call, [#PID<0.738.0>, {:sleep, 20}, 1]}},
   [
     {GenServer, :call, 3, [file: 'lib/gen_server.ex', line: 1023]},
     {Task.Supervised, :invoke_mfa, 2,
      [file: 'lib/task/supervised.ex', line: 90]},
     {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 226]}
   ]}`

They eventually get passed into `Exception.normalize`.

Reply:
@mitchellhenke none of those are exceptions though. They are all exits. And I don't think we should treat them as exceptions. If you really want those reported, then I would recommend enabling `capture_log_message` by default and then maybe set the default level to `:error`?